flang/lib/Lower/CallInterface.cpp

```diff
@@ -774,7 +775,8 @@ class Fortran::lower::CallInterfaceImpl {
   void handleExplicitDummy(
       const DummyCharacteristics *characteristics,
       const Fortran::evaluate::characteristics::DummyDataObject &obj,
-      const FortranEntity &entity) {
+      const FortranEntity &entity,
+      bool isBindC) {
```

Comment:
Looks like this needs to be run through clang-format.

Reply:
Fixed!